src/mod/applications/mod_curl/mod_curl.c

```diff
@@ -378,9 +378,14 @@ static char *print_json(switch_memory_pool_t *pool, http_data_t *http_data)
 				char *argv[3] = { 0 };
 				int argc;
 				if ((argc = switch_separate_string(header->data, ' ', argv, (sizeof(argv) / sizeof(argv[0]))))) {
-					if (argc > 2) {
-						cJSON_AddItemToObject(top, "version", cJSON_CreateString(argv[0]));
-						cJSON_AddItemToObject(top, "phrase", cJSON_CreateString(argv[2]));
+					if (argc >= 2) {
```

Comment:
This will not error if argc == 2 but `HTTP/1.1`.
I'd recommend adding something like an `else if (argc == 2 && argv[0] equals HTTP/2 )`